src/clr_bundle_add.c

```diff
@@ -45,6 +45,8 @@ static void print_help(const char *name)
 	printf("Help Options:\n");
 	printf("   -h, --help              Show help options\n");
 	printf("   -u, --url=[URL]         RFC-3986 encoded url for version string and content file downloads\n");
+	printf("   -c, --contenturl=[URL]  RFC-3986 encoded url for content file downloads\n");
+	printf("   -v, --versionurl=[URL]  RFC-3986 encoded url for version string download\n");
```

Comment:
@bradTpeters I agree that some commandline options should move to toplevel, but I would like to hold off on doing that until we rework the subcommand implementation so that all option parsing happens in swupd.c.